src/components/calculator.js

```diff
@@ -0,0 +1,28 @@
+const CalculatorElement = () => (
+  <div className="grid-container">
+    <div className="input-number-container">
+      <input className="input-number" />
+    </div>
+    <div>AC</div>
+    <div>+/-</div>
+    <div>%</div>
+    <div>/</div>
+    <div>7</div>
+    <div>8</div>
+    <div>9</div>
+    <div>X</div>
+    <div>4</div>
+    <div>5</div>
+    <div>6</div>
+    <div>-</div>
+    <div>1</div>
+    <div>2</div>
+    <div>3</div>
+    <div>+</div>
+    <div className="item-seventeen">0</div>
+    <div className="item-eighteen">.</div>
+    <div>=</div>
```

Comment:
- [OPTIONAL] I think it would be better to use buttons instead of `div` elements for each calculator button :+1: